mne/viz/tests/test_3d.py

```diff
@@ -318,4 +324,30 @@ def test_snapshot_brain_montage():
     assert_raises(TypeError, snapshot_brain_montage, fig, info)
 
 
+@requires_nilearn
+@spm_face.requires_spm_data
```

Comment:
how about adding it to testing dataset so it's actually tested on travis?
@Eric89GXL too much work re run recona-all and redo the testing dataset?
or maybe just rerun recon-all and copy the new ribbin mgz file?